lib/src/platform_text_field.dart

```diff
@@ -53,6 +53,54 @@ const BoxDecoration _kDefaultRoundedBorderDecoration = BoxDecoration(
   borderRadius: BorderRadius.all(Radius.circular(5.0)),
 );
 
+InputDecoration _inputDecorationWithHint(String hint, InputDecoration inputDecoration) {
+  return InputDecoration(
+    alignLabelWithHint: inputDecoration?.alignLabelWithHint ?? null,
+    border: inputDecoration?.border ?? null,
+    contentPadding: inputDecoration?.contentPadding ?? null,
+    counter: inputDecoration?.counter ?? null,
+    counterStyle: inputDecoration?.counterStyle ?? null,
+    counterText: inputDecoration?.counterText ?? null,
+    disabledBorder: inputDecoration?.disabledBorder ?? null,
+    enabled: inputDecoration?.enabled ?? null,
+    enabledBorder: inputDecoration?.enabledBorder ?? null,
+    errorBorder: inputDecoration?.errorBorder ?? null,
+    errorMaxLines: inputDecoration?.errorMaxLines ?? null,
+    errorStyle: inputDecoration?.errorStyle ?? null,
+    errorText: inputDecoration?.errorText ?? null,
+    fillColor: inputDecoration?.fillColor ?? null,
+    filled: inputDecoration?.filled ?? null,
+    floatingLabelBehavior: inputDecoration?.floatingLabelBehavior ?? null,
+    focusColor: inputDecoration?.focusColor ?? null,
+    focusedBorder: inputDecoration?.focusedBorder ?? null,
+    focusedErrorBorder: inputDecoration?.focusedErrorBorder ?? null,
+    hasFloatingPlaceholder: inputDecoration?.hasFloatingPlaceholder ?? null,
```

Comment:
```suggestion
    hasFloatingPlaceholder: inputDecoration?.hasFloatingPlaceholder ?? true,
```
Matches the InputDecoration of setting a variable in the constructor